docs/schemas/core/IfcProductExtension/Entities/IfcAlignment.md

```diff
@@ -14,90 +14,127 @@ An alignment is used to define a reference system to position elements mainly fo
 
-3. In the rail domain in most cases a cant layout is added to the horizontal layout to compensate a part of the unwanted lateral acceleration.
+3. In the rail domain, in most cases a cant layout is added to the horizontal layout to compensate a part of the unwanted lateral acceleration.
 
-4. In a final step the proposed layout is checked against a defined set of rules, formulas and thresholds to guarantee the conformance against the regulation.
+4. In a final step, the proposed layout is checked against a defined set of rules, formulas and thresholds to guarantee the conformance against the regulation.
 
 The sequence of steps might change from case to case and might be repeated one or more times to achieve the economic objectives and fulfill regulatory safety requirements.
 
+Contemporary alignment design almost always implements a 2.5 dimension approach. The resulting and documented geometry might be very precise or just good enough to meet safety thresholds. This depends on factors like priorities of the management, date of the design - existing alignments might have been designed more then 50 years ago - or software tools used. Working with legacy data in a high precision BIM model requires a good understanding of these factors.
 
-Contemporary alignment design almost always implements a 2.5 dimension approach.
+**Distinction between business logic and geometry definition**
 
-The resulting and documented geometry might be very precise or just good enough to meet safety thresholds. This depends on factors like priorities of the management, date of the design - existing alignments might have been designed more then 50 years ago - or software tools used. Working with legacy data in a high precision BIM model requires a good understanding of these factors.
+The alignment concept is organised in two parts. These two parts work together, but they can also be used and exchanged independently.
 
-**Distinction between business modeling and IFC core geometry**
+1. *Business logic* of alignment
+2. *Geometry definition* of alignment
 
-According to IFC modeling principles alignment entities are organised in two large parts. The two parts work together, but they can also be used independently from each other
+**Business logic:** the IFC schema allows to describe an alignment using terminology and concepts that are as close as possible to business ones. It allows to describe the layouts that make up the alignment (i.e. horizontal, vertical, cant), their segments' structure and attributes. Also, the *business logic* part provides the anchor point for domain specific properties, such as design speed or cant deficiency.
 
-1. Business aspects of alignment
-2. Representation with the IFC geometry resources
+**Geometry definition:** the IFC schema provides well established IFC geometric entities to represent the business concepts already defined. 
```

Comment:
```suggestion
**Geometry definition:** the IFC schema provides well established IFC geometric entities to represent the business concepts. 
```

Remove "already defined", because we said earlier: "can be used and exchanged independently"